### PR TITLE
Allow pasting images from clipboard

### PR DESCRIPTION
**Pull request by 0x0ade**
Small quality of life addition: Copy-pasting an image in the clipboard (from f.e. GIMP or by taking a screenshot) into the operator now behaves the same as when dropping an image into the operator.

The only change is that the file name is `clipboard-TIMESTAMP.TYPE`, where `TYPE` is determined from the clipboard item type and `TIMESTAMP` is `Date.now()`.

### CODE REVIEW

**Comment by composition4** [comment]
Have you tested yet to see if this is a Beaker support issue or a Rotonde support issue?

File: scripts/operator.js [hunk 1]
```diff
@@ -538,27 +539,45 @@ function Operator(el)
       var type = file.type;
 
       if (type === 'image/jpeg' || type === 'image/png' || type === 'image/gif') {
```
Comment (composition4) [inline scripts/operator.js]: I'd kinda like to see svg drag/drop and paste support here too. Is it possible or nah? I haven't experimented with them enough.

**Comment by 0x0ade** [comment]
SVG drag-n-drop isn't an issue :) I wonder if the Windows, macOS and Linux clipboards can handle SVG image data in a cross-application manner, though.

**Comment by 0x0ade** [comment]
@composition4 SVG drag-n-drop support added - we just weren't checking for it :)

The clipboard code is also more robust now, supporting cases where `items` contains multiple items (f.e. after copying an image from inside Chrome). Pasting SVG data should work with programs providing `image/svg+xml` data. Unfortunately, Inkscape isn't one of them and [an issue in Chromium's clipboard API implementation](https://bugs.chromium.org/p/chromium/issues/detail?id=504700) prevents f.e. dotgrid from storing SVGs in the clipboard as `image/svg+xml`. ~~Let's just store the SVG data as plain text and handle that...~~

**Comment by 0x0ade** [comment]
Pasting `text/svg+xml` (technically correct; practically something I just made up: `image/svg+xml` but in text form) now works just fine.

Semi-related: Currently working on a dotgrid patch that fully supports copying SVGs out of it. I'm still trying to get it working as a true `image/svg+xml`, but in the meantime, `text/svg+xml` works just fine for Rotonde. Something something Nataniev ecosystem? ;)

**Pull request merged**